src/atoms/Tag.js

```diff
@@ -16,14 +17,14 @@ const TagSpanItem = styled.span`
 	text-transform: uppercase;
 	color: ${getColor('grey')};
 `;
-const TagAItem = TagSpanItem.withComponent('a').extend.attrs({
+const TagAItem = withProps({
```

Comment:
I do not think this work, that could be way Shiny had a visual diff!
Need to check that up!